hamilton/plugins/jupyter_magic.py

```diff
@@ -87,74 +179,6 @@ def insert_cell_with_content():
     display(HTML(js_script))
 
 
-def find_all_hamilton_drivers_using_this_module(shell, module_name: str) -> list:
-    """Find all Hamilton drivers in the notebook that use the module `module_name`.
-
-    :param shell: the ipython shell object
-    :param module_name: the module name to search for
-    :return: the list of (driver variable name, drivers) that use the module
-    """
-    driver_instances = {
-        var_name: shell.user_ns[var_name]
-        for var_name in shell.user_ns
-        if isinstance(shell.user_ns[var_name], driver.Driver) and var_name != f"{module_name}_dr"
-    }
-    impacted_drivers = []
-    for var_name, dr in driver_instances.items():
-        for driver_module in dr.graph_modules:
-            if driver_module.__name__ == module_name:
-                impacted_drivers.append((var_name, dr))
-                break
-    return impacted_drivers
-
-
-def rebuild_drivers(shell, module_name: str, module_object: ModuleType, verbosity: int = 1) -> dict:
```

Comment:
what was the issue with this approach?

Reply:
I want to have the Jupyter magic owning the `Driver`